### PR TITLE
Deprecate Certificate expires_on in favor of expires_at

### PR DESCRIPTION
**Pull request by duduribeiro**
We deprecated expires_on attribute in our documentation and api in
favor of expires_at. This commit introcuces the new field and deprecates
the old one. It also updates to use the new fixtures.

### CODE REVIEW

File: lib/dnsimple/certificates.ex [hunk 1]
```diff
@@ -22,7 +22,7 @@ defmodule Dnsimple.Certificates do
       client = %Dnsimple.Client{access_token: "a1b2c3d4"}
       {:ok, response} = Dnsimple.Certificates.list_certificates(client, account_id = "1010", domain_id = "example.com")
       {:ok, response} = Dnsimple.Certificates.list_certificates(client, account_id = "1010", domain_id = "example.com", page: 2, per_page: 10)
-      {:ok, response} = Dnsimple.Certificates.list_certificates(client, account_id = "1010", domain_id = "example.com", sort: "expires_on:desc")
+      {:ok, response} = Dnsimple.Certificates.list_certificates(client, account_id = "1010", domain_id = "example.com", sort: "expiration:desc")
```
Comment (OleMchls) [inline lib/dnsimple/certificates.ex:25]: Have you verified this change? Seems unrelated

File: test/dnsimple/certificates_test.exs [hunk 2]
```diff
@@ -5,17 +5,16 @@ defmodule Dnsimple.CertificatesTest do
   @module Dnsimple.Certificates
   @client %Dnsimple.Client{access_token: "i-am-a-token", base_url: "https://api.dnsimple.test"}
   @account_id 1010
-  @domain_id "example.com"
```
Comment (OleMchls) [inline test/dnsimple/certificates_test.exs:8]: Is there a particular reason to move away from this [RFC defined](https://tools.ietf.org/html/rfc2606#section-3) test domain?

File: lib/dnsimple/certificates.ex (hunk 1 above)
Comment (duduribeiro) [reply lib/dnsimple/certificates.ex:25]: It is related. `expires_on` sorter exists, but it is deprecated in the app. We deprecated in favor of `expiration`

File: test/dnsimple/certificates_test.exs (hunk 2 above)
Comment (duduribeiro) [reply test/dnsimple/certificates_test.exs:8]: I preferred to change it to the domain used in the fixtures.
